Tasks/DotNetCoreInstallerV0/externals/install-dotnet.sh

```diff
@@ -654,7 +658,19 @@ install_dotnet() {
     say_verbose "Zip path: $zip_path"
 
     say "Downloading link: $download_link"
-    download "$download_link" $zip_path || download_failed=true
+    download "$download_link" $zip_path || fallback_download_failed=true
+
+    say_verbose "download_failed: $download_failed"
+    say_verbose "valid_legacy_download_link: $valid_legacy_download_link"
```

Comment:
I think this should move down to above "#  if the download fails, download the legacy_download_link"

Reply:
updated